### ISSUE TITLE
Scnn1a_473845048_m.swc or Scnn1a_473845048_m?

### ISSUE THREAD

**Issue by pgleeson** [AllenInstitute/sonata#40]
@kaeldai The example https://github.com/AllenInstitute/sonata/blob/master/examples/9_cells/build_network.py requires Scnn1a_473845048_m to be changed to Scnn1a_473845048_m.swc (i.e. adding swc suffix) to run, but https://github.com/AllenInstitute/sonata/blob/master/examples/5_cells_iclamp/build_network.py works fine without. Could you check this? I'm using the latest BMTK from the develop branch.

Also, I'm getting the following error when running https://github.com/AllenInstitute/sonata/blob/master/examples/5_cells_iclamp/run_bionet.py

```
> python run_bionet.py
NEURON -- VERSION 7.5 master (6b4c19f) 2017-09-25
Duke, Yale, and the BlueBrain Project -- Copyright 1984-2016
See http://neuron.yale.edu/neuron/credits

2018-10-01 18:30:59,689 [INFO] Created log file
loading membrane mechanisms from ./../shared_components/mechanisms/x86_64/.libs/libnrnmech.so
Additional mechanisms from files
 modfiles//CaDynamics.mod modfiles//Ca_HVA.mod modfiles//Ca_LVA.mod modfiles//Ih.mod modfiles//Im.mod modfiles//Im_v2.mod modfiles//Kd.mod modfiles//K_P.mod modfiles//K_T.mod modfiles//Kv2like.mod modfiles//Kv3_1.mod modfiles//Nap.mod modfiles//NaTa.mod modfiles//NaTs.mod modfiles//NaV.mod modfiles//SK.mod modfiles//vecevent.mod
2018-10-01 18:30:59,743 [INFO] Building cells.
2018-10-01 18:31:00,110 [INFO] Building recurrent connections
Traceback (most recent call last):
  File "run_bionet.py", line 17, in <module>
    run('config.json')
  File "run_bionet.py", line 10, in run
    sim.run()
  File "/usr/local/lib/python2.7/dist-packages/bmtk-0.0.6-py2.7.egg/bmtk/simulator/bionet/biosimulator.py", line 210, in run
    mod.initialize(self)
  File "/usr/local/lib/python2.7/dist-packages/bmtk-0.0.6-py2.7.egg/bmtk/simulator/bionet/modules/ecp.py", line 169, in initialize
    self._calculate_ecp(sim)
  File "/usr/local/lib/python2.7/dist-packages/bmtk-0.0.6-py2.7.egg/bmtk/simulator/bionet/modules/ecp.py", line 105, in _calculate_ecp
    self._rel = RecXElectrode(self._positions_file)
  File "/usr/local/lib/python2.7/dist-packages/bmtk-0.0.6-py2.7.egg/bmtk/simulator/bionet/modules/ecp.py", line 230, in __init__
    el_df = pd.read_csv(electrode_file, sep=' ')
  File "/usr/local/lib/python2.7/dist-packages/pandas/io/parsers.py", line 645, in parser_f
    return _read(filepath_or_buffer, kwds)
  File "/usr/local/lib/python2.7/dist-packages/pandas/io/parsers.py", line 388, in _read
    parser = TextFileReader(filepath_or_buffer, **kwds)
  File "/usr/local/lib/python2.7/dist-packages/pandas/io/parsers.py", line 729, in __init__
    self._make_engine(self.engine)
  File "/usr/local/lib/python2.7/dist-packages/pandas/io/parsers.py", line 922, in _make_engine
    self._engine = CParserWrapper(self.f, **self.options)
  File "/usr/local/lib/python2.7/dist-packages/pandas/io/parsers.py", line 1389, in __init__
    self._reader = _parser.TextReader(src, **kwds)
  File "pandas/parser.pyx", line 373, in pandas.parser.TextReader.__cinit__ (pandas/parser.c:4175)
  File "pandas/parser.pyx", line 667, in pandas.parser.TextReader._setup_parser_source (pandas/parser.c:8440)
IOError: File $COMPONENT_DIR/recXelectrodes/linear_electrode.csv does not exist

```

**Comment by hernando** [comment]
I don't know the details about those scripts, but morphology names mustn't have the .swc extension as per the current spec.

**Comment by kaeldai** [comment]
In the 9 cells build_network.py script, it needs to parse the swc file to know what segments to place the synapses. So I changed the build script to resolve the "morphology" column value (i.e. append .swc).  The 5 cell current clamp example doesn't need this because no synapses are created, just injections into the soma.

Also, in the 5-cell simulation_config.json I needed to change $COMPONENT_DIR to $COMPONENT**S**_DIR. It looks like I didn't catch these fixes on my last commit. Hopefully this time it sticks.
